README.md

```diff
@@ -39,7 +39,7 @@ $ npm install -g @vertexvis/cli
 $ vertex COMMAND
 running command...
 $ vertex (-v|--version|version)
-@vertexvis/cli/0.16.1 darwin-x64 node-v16.14.0
+@vertexvis/cli/0.16.2 darwin-x64 node-v14.17.3
```

Comment:
Why are we using an older version of node?

Reply:
😱  How did that get in there???

Reply:
oh, this README.md pulls this from my local version

Reply:
@amvertex updated my local node version to the latest lts and re-ran.

Reply:
Perfect!